main/src/main/java/org/qbicc/main/Main.java

```diff
@@ -378,7 +377,6 @@ DiagnosticContext call0(BaseDiagnosticContext initialContext) {
                                 }
                                 builder.addBuilderFactory(Phase.LOWER, BuilderStage.INTEGRITY, LowerVerificationBasicBlockBuilder::new);
 
-                                builder.addPreHook(Phase.GENERATE, new HeapSerializer());
```

Comment:
BuildTimeHeap now generates the Data declarations incrementally into the right Section.  So we don't need an extra hook just for that.